src/components/AllServicesDropdown/AllServicesGallery.tsx

```diff
@@ -13,10 +13,10 @@ const AllServicesGallery = ({ selectedService }: AllServicesGalleryProps) => {
   const sections: AllServicesGroup[] = [];
   const links: AllServicesLink[] = [];
   selectedService.links.forEach((link) => {
-    if (isAllServicesGroup(link)) {
-      sections.push(link);
+    if (isAllServicesGroup(link as AllServicesGroup)) {
```

Comment:
We can change the `isAllServicesGroup` to accept `any` or `unknow` rather than typecasting the arguments. it kind of defeats the purpose of the typeguard.

Reply:
Good point!